content/en/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -101,7 +101,7 @@ in some namespaces.  However namespace resources are not themselves in a namespa
 And low-level resources, such as [nodes](/docs/admin/node) and
 persistentVolumes, are not in any namespace.
 
-To see which Kubernetes resources are and aren't in a namespace:
+To see which Kubernetes resources are and aren't in a namespace, only support over v1.11.0:
```

Comment:
1.11.0 is a "long" history. We are about to deprecate docs for 1.11.0. Does it still make sense to mention an old version in the new doc?

Reply:
I know it, but in production env, i think `1.10.0` still exist, comment version is exact.